tests/auto/main_validator.test.js

```
@@ -0,0 +1,627 @@
+require = originalRequire;
+
+describe('MainValidator', () => {
+  let validator;
+  let tempDir;
+
+  beforeEach(() => {
+    validator = new MainValidator();
+    tempDir = tmpDir({ unsafeCleanup: true });
+    
+    // Reset all stubs
+    Object.values(mockSecurityValidator).forEach(stub => stub.resetHistory && stub.resetHistory());
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)